### DISCUSSION TITLE
Getting event from Ajax Call or Refresh whole calender

### DISCUSSION THREAD

**Discussion by SweAdeel** [vkurko/calendar#366] [Q&A]
Hello Mate

how to get the events from ajax call or calling event in function we can recall or refresh calender for any new events

if there any function/option for this ?

kindly reply

Thanks
Cheers !

**Comment by vkurko** [comment]
Hello,

Please see [eventSources](https://github.com/vkurko/calendar#eventsources) option and [refetchEvents](https://github.com/vkurko/calendar#refetchevents) method.
